aries-site/src/examples/cardPreviews/emptystate.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { Box, Button, Heading, Paragraph } from 'grommet';
+import { CircleInformation } from 'grommet-icons';
+
+export const EmptyStatePreview = () => (
+  <Box gap="small" align="center" flex={false}>
```

Comment:
Can we use `EmptyState` from aries-core directly so this adapts overtime if designs change?

For `action` can you apply `tabIndex={-1}` to the button so it's not tabbable.

Also, we're getting a failing test since EmptyState uses a semantic heading -- I think it might need to be `h3`, but can you double check with WAVE plugin?

Reply:
yes, I had that originally but the text looked super big but I can change it back.